Firebase/Database/Core/FServerValues.m

```diff
@@ -20,37 +20,118 @@
 #import "FLeafNode.h"
 #import "FSnapshotUtilities.h"
 
+const NSString *kTimestamp = @"timestamp";
+const NSString *kIncrement = @"increment";
+
+BOOL longCanHold(NSNumber *num) {
```

Comment:
Not a huge fan of the name, but it's probably ok :)  I'd suggest something like `canBeRepresentedAsLong`.

Optional.